data/config-schema/src/main/resources/net/corda/schema/configuration/sandbox/1.0/corda.sandbox.json

```diff
@@ -5,16 +5,63 @@
   "description": "Configuration schema for the sandbox subsection.",
   "type": "object",
   "properties": {
-    "cache": {
-      "description": "Settings for sandbox caching",
+    "flow": {
+      "description": "Settings for flow sandbox",
       "type": "object",
       "default": {},
       "properties": {
-        "size": {
-          "description": "The maximum number cached sandboxes.",
-          "type": "integer",
-          "minimum": 0,
-          "default": 2
+        "cache": {
+          "description": "Settings for flow sandbox caching",
+          "type": "object",
+          "default": {},
+          "properties": {
+            "size": {
+              "description": "The maximum number cached flow sandboxes.",
+              "type": "integer",
+              "minimum": 0,
+              "default": 2
```

Comment:
Sorry to be a pain.
Can we set all defaults to 10? It's a bit arbitrary, but I hope it's a more appropriate level and we can review again after we see the impact on the soaktests and/or perf testing.
FYI @JamesHR3 @davidcurrie

Reply:
Cool, I've changed that.  (sorry, I acidentally amended the commit so a bit of history is lost of the PR)